Dockerfile

```diff
@@ -14,36 +14,17 @@ RUN \
     log () { printf "\n%s\t%s\n\n" "$(date '+%F %X %z')" "$*" ; } ; \
     # Attempt to grab the major version from the tag \
     DVER=$(egrep -o '[0-9][\.0-9]*$' <<< "$IMAGE_BASE" | cut -d. -f1); \
-    if  [[ $DVER == 7 ]]; then \
-       YUM_PKG_NAME="yum-plugin-priorities"; \
-       yum-config-manager \
-         --setopt=skip_missing_names_on_install=False \
-         --setopt=skip_missing_names_on_update=False \
-         --save > /dev/null; \
-    else \
-       YUM_PKG_NAME="yum-utils"; \
-    fi && \
     log "Updating OS YUM cache" && time \
     yum makecache && \
     log "Updating OS" && time \
     yum distro-sync -y && \
-    if [[ $OSG_RELEASE =~ ^[0-9][0-9]$ ]]; then \
-       OSG_URL=https://repo.opensciencegrid.org/osg/${OSG_RELEASE}-main/osg-${OSG_RELEASE}-main-el${DVER}-release-latest.rpm; \
-    else \
-       OSG_URL=https://repo.opensciencegrid.org/osg/${OSG_RELEASE}/osg-${OSG_RELEASE}-el${DVER}-release-latest.rpm; \
-    fi && \
+    OSG_URL=https://repo.opensciencegrid.org/osg/${OSG_RELEASE}-main/osg-${OSG_RELEASE}-main-el${DVER}-release-latest.rpm && \
     log "Installing EPEL/OSG repo packages" && time \
     yum -y install $OSG_URL \
                    epel-release \
-                   $YUM_PKG_NAME && \
```

Comment:
Is `yum-utils` no longer needed in el8+?

Reply:
It's hard to see in the diff but I do actually install it